### DISCUSSION TITLE
Column heading and horizontal scrollbar in list view

### DISCUSSION THREAD

**Discussion by dkanthofer** [payloadcms/payload#3876] [Feature Requests & Ideas]
Currently, when you scroll down in the list view of a collection, the column headings will scroll out of the viewport. 
It would be great, if there was a way to make the headings sticky, so you can always see them while scrolling through the list.

Also, the horizontal scrollbar, that appears when you have a lot of columns, is only visible when you scroll down the whole list. This scrollbar should always be visible as long as you have too many columns to fit your browser window's width.

**Comment by DanielRolfs** [comment]
that would be great! My colleague who is maintaintaining the content asked me today if I can make the tabs sticky so she can easily jump to the next tab once the finished a tab. Currently she need to scroll manually all the way up.
